youtube_dl/extractor/vier.py

```diff
@@ -38,13 +42,41 @@ class VierIE(InfoExtractor):
             'skip_download': True,
         },
     }, {
+        'url': 'http://www.vier.be/janigaat/videos/jani-gaat-naar-tokio-aflevering-4/2674839',
+        'info_dict': {
+            'id': '2674839',
+            'display_id': 'jani-gaat-naar-tokio-aflevering-4',
+            'ext': 'mp4',
+            'title': 'Jani gaat naar Tokio - Aflevering 4',
+            'description': 'Bekijk hier de volledige vierde aflevering van het 2de seizoen van Jani gaat...',
+        },
+        'params': {
+            # m3u8 download
+            'skip_download': True,
+        }
+    }, {
         'url': 'http://www.vier.be/planb/videos/mieren-herders-van-de-bladluizen',
         'only_matching': True,
     }, {
         'url': 'http://www.vier.be/video/v3/embed/16129',
         'only_matching': True,
     }]
 
+    def _login(self, site):
+        username, password = self._get_login_info()
+        if username is None or password is None:
+            self.raise_login_required()
```

Comment:
There could be a fallback to video/embed URL and extraction without metadata with a warning to use a credentials for correct metadata printed.